webapp/lib/view.dart

```diff
@@ -562,8 +552,61 @@ class BatchRepliesConfigurationView extends PackageConfiguratorView {
 
   void _addTag(String tag, Element tagList) {
     // TODO: call controller.command()
-    tagList.lastChild.remove();
+    tagList.lastChild.lastChild.remove();
     tagList.children.last.insertAdjacentElement('beforebegin', new TagView(tag, tag, TagStyle.Normal).renderElement);
+    model.tags.removeWhere((tg) => tg == tag);
+  }
+
+  DivElement _addSuggestedResponseEntry([Map suggestedResponse]) {
+    return new DivElement()
+      ..classes.add('conversation-response')
+      ..append(
+        new ButtonElement()
+          ..classes.add('button-remove-conversation-responses')
+          ..text = 'x'
+          ..onClick.listen((event) => (event.target as Element).parent.remove()) // TODO: call controller.command
+      )
+      ..append(
+        new ParagraphElement()
+          ..classes.add('conversation-response__language')
+          ..text = suggestedResponse != null ? suggestedResponse['messages'][0] : ''
+          ..contentEditable = 'true'
+      )
+      ..append(
+        new ParagraphElement()
+          ..classes.add('conversation-response__language')
+          ..text = suggestedResponse != null ? suggestedResponse['messages'][1] : ''
+          ..contentEditable = 'true'
+      )
+      ..append(
+        DivElement()
+          ..classes.add('conversation-response__reviewed')
+          ..append(
+            new CheckboxInputElement()
+                ..classes.add('conversation-response__reviewed-state')
+                ..checked = suggestedResponse != null ? suggestedResponse['reviewed'] : false
+                ..onClick.listen((event) => _reviewSuggestedReplies(event))
+          )
+          ..append(
+            new ParagraphElement()
+            ..classes.add('conversation-response__reviewed-description')
+            ..text = suggestedResponse != null ? '${suggestedResponse['reviewed-by']}, ${suggestedResponse['reviewed-date']}' : ','
```

Comment:
I think we shouldn't really show the comma if there's no data to show
```suggestion
            ..text = suggestedResponse != null ? '${suggestedResponse['reviewed-by']}, ${suggestedResponse['reviewed-date']}' : ''
```